content_script.js

```diff
@@ -1,32 +1,33 @@
-/** 
+/**
   * Class Schedule to ICS
-  * (c) 2013 Keanu Lee
+  * (c) 2013-2015 Keanu Lee
+  * (c) 2015 Baraa Hamodi
```

Comment:
You can contribute to the code, but its copyright can't be assigned to two people. If you want to leave your name in comments, do something along the lines of `With contributions from ...`